spring-batch-core/src/test/java/org/springframework/batch/core/JobExecutionTests.java

```diff
@@ -34,8 +35,13 @@
  */
 public class JobExecutionTests {
 
-	private JobExecution execution = new JobExecution(new JobInstance(new Long(11), "foo"),
-			new Long(12), new JobParameters(), null);
+	private JobExecution execution;
+
+	@Before
```

Comment:
we can keep the inline initialization in this PR. I usually don't change (or polish) and add code in the same change set (unless it is necessary) in order to see what really fixes the issue. Moving the initialization to a `@Before` method (which is a good idea) is better done in a polishing commit.